db/ddl.sql

```diff
@@ -135,3 +135,4 @@ CREATE TABLE orbital_responses (
 CREATE INDEX index_orbital_responses_kb_payment_id_kb_tenant_id ON orbital_responses(kb_payment_id, kb_tenant_id);
 CREATE INDEX index_orbital_responses_kb_payment_transaction_id_kb_tenant_id ON orbital_responses(kb_payment_transaction_id, kb_tenant_id);
 CREATE INDEX index_orbital_responses_params_tx_ref_num_kb_tenant_id ON orbital_responses(params_tx_ref_num, kb_tenant_id);
+CREATE INDEX index_orbital_responses_success ON orbital_responses(success);
```

Comment:
Shouldn't we have a composite index with the `kb_tenant_id` or is this search cross tenant?

Reply:
This is cross tenant: https://github.com/killbill/killbill-plugin-framework-ruby/pull/68/files.

But based on the conversation at https://github.com/killbill/killbill-plugin-framework-ruby/issues/67, I'm guessing @Ecsiral will tweak the implementation instead (probably with something similar to https://github.com/killbill/killbill/blob/2d66e2ac44764618d1dca2910c1abd65ef2c3149/util/src/main/java/org/killbill/billing/util/entity/dao/DefaultPaginationSqlDaoHelper.java#L79-L83)?